docs/docsite/rst/user_guide/playbooks_vault.rst

```diff
@@ -7,26 +7,33 @@ Using Vault in playbooks
 
 The "Vault" is a feature of Ansible that allows you to keep sensitive data such as passwords or keys in encrypted files, rather than as plaintext in playbooks or roles. These vault files can then be distributed or placed in source control.
 
-To enable this feature, a command line tool, :ref:`ansible-vault` is used to edit files, and a command line flag :option:`--ask-vault-pass <ansible-vault --ask-vault-pass>` or :option:`--vault-password-file <ansible-vault --vault-password-file>` is used. You can also modify your ``ansible.cfg`` file to specify the location of a password file or configure Ansible to always prompt for the password. These options require no command line flag usage.
+To enable this feature, a command line tool, :ref:`ansible-vault` is used to edit files and the command line flag :option:`--vault-id <ansible-playbook --vault-id>` cli option is used. You can also modify your ``ansible.cfg`` file to specify the location of a password file or configure Ansible to always prompt for the password. These options require no command line flag usage.
 
 For best practices advice, refer to :ref:`best_practices_for_variables_and_vaults`.
 
+.. note::
+    The :option:`--vault-id <ansible-playbook --vault-id>` flag is only avalibe in Ansible 2.4 or later.
+    Prior to Ansible 2.4, flags :option:`--ask-vault-pass <ansible-vault --ask-vault-pass>` or
+    :option:`--vault-password-file <ansible-vault --vault-password-file>` are used.
```

Comment:
these are still usable if you dont want to use the 'id' system, i would leave them in the main text and maybe clarify this in the notes